File: CAPITULO 02/Java/ex10.java
import java.io.*;
import java.util.*;
import java.time.*;
public class ex10{
	public static void main(String[]args){
		int dia1,mes1,ano1,dia2,mes2,ano2;
		Scanner pt = new Scanner (System.in);
		System.out.println("Digite duas datas quaisquer distintas");
		System.out.println("Digita o dia :");
		dia1 = pt.nextInt();
		System.out.println("Digite o mes :");
		mes1 = pt.nextInt();
		System.out.println("Digite o ano :");
		ano1 = pt.nextInt();
		System.out.println("=============Digite outra data");
		System.out.println("Digita o dia :");
		dia2 = pt.nextInt();
		System.out.println("Digite o mes :");
		mes2 = pt.nextInt();
		System.out.println("Digite o ano :");
		ano2 = pt.nextInt();
		if(dia1 > dia2 || mes1>mes2 || ano1>ano2){
			System.out.println("Primeira data digitada maior");
		}
		else{
			System.out.println("Segunda data digitada maior");
		}
		
	}
}

File: CAPITULO 02/C++/ex10.c
#include <stdio.h>
#include <locale.h>
int main(){
int dia1,mes1,ano1,dia2,mes2,ano2;
printf("Digite duas datas distintas quaisquer\n");
printf("Digite primeiro o dia :");
scanf("%d%*c",&dia1);
printf("Digite o mes :");
scanf("%d%*c",&mes1);
printf("Digite o ano :");
scanf("%d%*c",&ano1);
printf("Digite outra data\n");
printf("Digite primeiro o dia :");
scanf("%d%*c",&dia2);
printf("Digite o mes :");
scanf("%d%*c",&mes2);
printf("Digite o ano ");
scanf("%d%*c",&ano2);
if(dia1>dia2 || mes1>mes2 || ano1>ano2)
    printf("Primeira data digitada maior");
else
    printf("Segunda data digita maior");
    system("pause");
return 0;
}
